e3db/src/main/java/com/tozny/e3db/ErrorResult.java

```diff
@@ -71,16 +71,16 @@ public R asValue() {
   /**
    * Returns any E3DB-specific error that occurred.
    *
-   * <p>This method will return {@code null} if a general exception
+   * <p>This method will return the underlying exception if a general exception
    * occurred during the operation; otherwise, it will return a specific E3DB error.
    *
-   * @return The  error, if any.
+   * @return The error, if any.
    */
   public E3DBException error() {
```

Comment:
I don't think this will compile anymore because return types won't match. If this is the outcome we want then we would just want the return type to be `throwable`. If that is what we are going to do then just return the error without the logic as well.